elliottlib/bzutil.py

```diff
@@ -50,7 +50,7 @@ def search_for_bugs(bz_data, status, search_filter='default', filter_out_securit
     if verbose:
         click.echo(query_url)
     
-    return _perform_query(bzapi, query_url)
+    return _perform_query(bzapi, query_url, include_fields=['id', 'status', 'summary', 'creation_time', 'cf_pm_score', 'component'])
```

Comment:
Does this slow the query down? I'm thinking of @ scale when we're finding like 500+ bugs to attach. If it does add a penalty we might only want to add those additional fields if report is requested.

Obvious followup: "Why not yaml?"

Reply:
I actually don't know if that affects the speed or not. The cli tool seems to pull all the fields in by default. But, the fact that this is a feature makes me think it probably has some effect, and it is easy enough to make this a conditional.

Reply:
I just did some quick checks and the speed was the same with and without `report` :+1: